### DISCUSSION TITLE
Does EC(Erasure code) schema work for any object size?

### DISCUSSION THREAD

**Discussion by gennady-voronkov** [minio/minio#13985] [Q&A] [answered]
Does EC(Erasure code) schema work for any object size?
I did some test with six object size: 10K, 100K, 1M, 10M, 100M 200M
where first three objects size have small perfomance

**Comment by harshavardhana** [comment, marked as answer]
Yes

**Comment by gennady-voronkov** [comment]
What is chunk size? and chunk size is related with EC schema?

**Reply by gennady-voronkov** [reply]
ok, thanks a lot
